examples/Planning/ForecastReach.cs

```diff
@@ -0,0 +1,313 @@
+        /// <param name="locationId">Location ID from somewhere returned by ListPlannableLocations.</param>
+        public void ShowPlannableProducts(ReachPlanServiceClient reachPlanService, string locationId)
+        {
+            ListPlannableProductsRequest request = new ListPlannableProductsRequest()
+            {
+                PlannableLocationId = locationId
+            };
+            ListPlannableProductsResponse response = reachPlanService.ListPlannableProducts(request);
+
+            Console.WriteLine($"Plannable Products for location {locationId}:");
+            foreach (ProductMetadata product in response.ProductMetadata)
+            {
+                Console.WriteLine($"{product.PlannableProductCode}:");
+                Console.WriteLine("Age Ranges:");
+                foreach (var ageRange in product.PlannableTargeting.AgeRanges)
+                {
+                    Console.WriteLine($"\t- {ageRange}");
+                }
+                Console.WriteLine("Genders:");
+                foreach (var gender in product.PlannableTargeting.Genders)
+                {
+                    Console.WriteLine($"\t- {gender.Type}");
+                }
+                Console.WriteLine("Devices:");
+                foreach (var device in product.PlannableTargeting.Devices)
+                {
+                    Console.WriteLine($"\t- {device.Type}");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Create a base request to generate a reach forecast.
+        /// </summary>
+        /// <param name="customerId">The customer ID for the reach forecast.</param>
+        /// <param name="productMix">The product mix for the reach forecast.</param>
+        /// <param name="locationId">Location ID from somewhere returned by ListPlannableLocations.</param>
+        /// <param name="currencyCode">Three-character ISO 4217 currency code.</param>
+        public GenerateReachForecastRequest BuildReachRequest(
+                   string customerId,
+                   List<PlannedProduct> productMix,
+                   string locationId,
+                   string currencyCode)
+        {
+            CampaignDuration duration = new CampaignDuration()
+            {
+                DurationInDays = 30
+            };
+
+            GenderInfo[] genders = new GenderInfo[]
+            {
+                new GenderInfo() { Type = GenderType.Female },
+                new GenderInfo() { Type = GenderType.Male }
+            };
+
+            DeviceInfo[] devices = new DeviceInfo[]
+            {
+                new DeviceInfo() { Type = Device.Desktop },
+                new DeviceInfo() { Type = Device.Mobile },
+                new DeviceInfo() { Type = Device.Tablet }
+            };
+
+            Targeting targeting = new Targeting()
+            {
+                PlannableLocationId = locationId,
+                AgeRange = ReachPlanAgeRange.AgeRange1865Up,
```

Comment:
This seems very broad, no? Can we use a narrower age range in this example?

Reply:
We're using the same defaults as we do for Reach Planner in the UI (I also updated duration to  match that).